### PR TITLE
Add network to influxdb. Fix #345

### LINKED ISSUE #345: No connection to InfluxDB
The influx docker-compose.yaml needs the same network change (‘docker_mainflux-base-net’) in order to be able to connect to it again.

### PR DESCRIPTION
**Pull request by ks156**
Signed-off-by: Paul RATHGEB <paul.rathgeb@skynet.be>

### CODE REVIEW

**Comment by drasko** [comment]
LGTM, but maybe generalizing the `networks` for all the service in this influxdb docker-compose add-on as @RubenN  suggested would be better solution. I guess for Grafana it is not necessary, because only Influx "links" to the services from other docker-compose file, but in more generic case there will be more services that will need this connection.

@dusanb94 @anovakovic01 @nmarcetic please review.

**Comment by dborovcanin** [comment]
@drasko I share your opinion. I think networking should be handled the same way as in the other services, which is the approach @RubenN suggested.

**Comment by drasko** [comment]
@ks156 can you please refine your PR to use approach that @RubenN suggested.

**Comment by ks156** [comment]
@drasko Synced with master, and added the approach of @RubenN

**Review by drasko — APPROVED** [review]
LGTM

File: docker/addons/influxdb/docker-compose.yml [hunk 1]
```diff
@@ -8,6 +8,10 @@
 
 version: "3"
 
+networks:
```
Comment (drasko) [inline docker/addons/influxdb/docker-compose.yml]: Actually - should be `external` - like in other `writers`: https://github.com/mainflux/mainflux/blob/master/docker/addons/mongodb-writer/docker-compose.yml#L12

**Review by drasko — CHANGES_REQUESTED** [review]

File: docker/addons/influxdb/docker-compose.yml (hunk 1 above)
Comment (ks156) [reply docker/addons/influxdb/docker-compose.yml]: Changes applied.

Comment (drasko) [reply docker/addons/influxdb/docker-compose.yml]: Actually, `docker_` prefix is missing. I think it is important for Kubernetes.

**Review by drasko — CHANGES_REQUESTED** [review]

**Comment by ks156** [comment]
#361 replaces this PR

**Pull request closed without merging**